src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -42,6 +44,8 @@ fluent-toolbar[orientation=horizontal] {
     --error: #F8040A !important;
     --foreground-subtext-rest: #8D8D8D;
     --foreground-subtext-hover: #A1A1A1;
+    --error-ui-foreground-color: var(--foreground-on-accent-rest);
+    --error-ui-accent-foreground-color: var(--accent-base-color);
```

Comment:
You don't generally want to use `--accent-base-color` for much - it's used to calculate the accent palette, but doesn't have any guarantees about contrast, etc. `--accent-foreground-rest` as used in the light theme would be more appropriate.

Reply:
accent-foreground-rest does not work here, as it does not provide appropriate contrast